tests/stress/stress_test.go

```diff
@@ -13,19 +13,21 @@ type StressTest struct {
 	Layer0    *clients.Layer0TestClient
 }
 
-func NewStressTest(t *testing.T, dir string, vars map[string]string) *StressTest {
+func NewStressTest(t *testing.B, dir string, vars map[string]string) *StressTest {
 	if vars == nil {
 		vars = map[string]string{}
 	}
 
 	vars["endpoint"] = config.APIEndpoint()
 	vars["token"] = config.AuthToken()
 
-	tfContext := tftest.NewTestContext(t,
+	tfContext := tftest.NewTestContext(
+		t,
 		tftest.Dir(dir),
 		tftest.Vars(vars),
 		tftest.DryRun(*dry),
-		tftest.Log(log))
+		tftest.Log(t),
```

Comment:
This shouldn't change - should still be `tftest.Log(log)` since that's how the `-debug` flag gets used

Reply:
✅